sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -510,7 +548,7 @@ export class InstitutionController extends BaseController {
    */
   @AllowAuthorizedParty(AuthorizedParties.aest)
   @Groups(UserGroups.AESTUser)
-  @Get("/:institutionId")
+  @Get("/:institutionId/basic-details")
```

Comment:
I guess this is the API conflict ? Thanks for making the change.

Reply:
yup @dheepak-aot